src/components/dashboard/sidebar/SubsidiesSummary.tsx

```diff
@@ -120,7 +121,7 @@ const SubsidiesSummary = ({
 
   return (
     <Card
-      className={classNames('mb-5', { 'col-8 border-0 shadow-none': programProgressPage })}
+      className={classNames(`mb-${hasApplicableLearnerCredit ? '5' : '0'}`, { 'col-8 border-0 shadow-none': programProgressPage })}
```

Comment:
nit: might simplify by using `classNames` to handle the conditional/dynamic class name, e.g.

```js
classNames({
  'mb-5:' hasApplicableLearnerCredit,
  'mb-0': !hasApplicableLearnerCredit,
  'col-8 border-0 shadow-none': programProgressPage,
})
```